# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/html/cross-origin-opener-policy/tentative/restrict-properties/iframe-popup-to-un.https.html.ini
# Commit: 425cbe3c033b
# Full Hash: 425cbe3c033bfc25795979513ebd2bb9fda7338d
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2024-11-28 21:54:11
# Regressor Bug: 1933829
# File Overlap Count: 1
# Description:
#   Bug 1933829 - [wpt-sync] Update web-platform-tests to 2d5d222739fdf91743f8cbc58094976a613194e7, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 2d5d222739fdf91743f8cbc58094976a613194e7
# ==============================================================================

diff -r 732bcd7ac971 -r 425cbe3c033b testing/web-platform/meta/html/cross-origin-opener-policy/tentative/restrict-properties/iframe-popup-to-un.https.html.ini
--- a/testing/web-platform/meta/html/cross-origin-opener-policy/tentative/restrict-properties/iframe-popup-to-un.https.html.ini	Wed Nov 27 22:05:28 2024 +0000
+++ b/testing/web-platform/meta/html/cross-origin-opener-policy/tentative/restrict-properties/iframe-popup-to-un.https.html.ini	Wed Nov 27 22:05:56 2024 +0000
@@ -41,6 +41,8 @@
 
 
 [iframe-popup-to-un.https.html?7-8]
+  expected:
+    if not sessionHistoryInParent and not debug: [OK, ERROR]
   [COOP: restrict-properties to popup COOP: unsafe-none via an iframe, with same origin iframe, cross origin popup with window_open]
     expected: FAIL
 
@@ -75,7 +77,8 @@
 
 [iframe-popup-to-un.https.html?3-4]
   expected:
-    if (os == "android") and not debug: [OK, TIMEOUT]
+    if (os == "android") and not debug and sessionHistoryInParent: [OK, TIMEOUT]
+    if (os == "android") and not debug and not sessionHistoryInParent: [OK, ERROR, TIMEOUT]
   [COOP: restrict-properties to popup COOP: unsafe-none via an iframe, with cross origin iframe, same origin popup with window_open]
     expected: FAIL
 